docs/kqueen.rst

```diff
@@ -152,26 +152,235 @@ in the configuration file, set the environment variable matching the KQUEEN_<con
+   .. code-block:: bash
+
+      # Domain name for service. Should be equal with name in generated ssl-certificate
+      NGINX_VHOSTNAME=demo.kqueen.net
+      # Directory path for certificates in container.Finally it look like $NGINX_SSL_CERTIFICATE_DIR/$NGINX_VHOSTNAME
+      NGINX_SSL_CERTIFICATE_DIR=/mnt/letsencrypt
+
+2. Open the docker-compose.production.yml file for editing.
+3. Check proxy service configuration in docker-compose.production.yml. Pay attention on following variables:
+
+  1. ``VHOSTNAME`` - domain name for Kqueen service. Should be equal with domain name in generated certificates. Default: Using variable from .env file, named NGINX_VHOSTNAME
+  2. ``SSL_CERTIFICATE_DIR`` - Mapped directory for certificates forwarding into docker container. Default: Using variable from .env file, named NGINX_SSL_CERTIFICATE_DIR/NGINX_VHOSTNAME
+  3. ``SSL_CERTIFICATE_PATH`` - Path for cert+key certificate. Default: ``$SSL_CERTIFICATE_DIR/fullchain.cer``
+  4. ``SSL_CERTIFICATE_KEY_PATH`` -  Path for certificate key. Default: ``$SSL_CERTIFICATE_DIR/$VHOSTNAME.key``
+  5. ``SSL_TRUSTED_CERTIFICATE_PATH`` - Path for certificate only. Default: ``$SSL_CERTIFICATE_DIR/ca.cer``
+
+.. note::
+
+   Check that local certificates naming equal to defined in variables.
+
+4. Map volumes with certificates. Pay attention that destination path should be equal with SSL_CERTIFICATE_DIR. Example:
+
+   .. code-block:: yaml
+
+      volumes:
+        - /your/local/cert/storage/kqueen/certs/:${NGINX_SSL_CERTIFICATE_DIR}/${NGINX_VHOSTNAME}:ro
+
+5. Build proxy service image:
+
+   .. code-block:: bash
+
+      docker-compose -f docker-compose.production.yml build --no-cache
+
+6. Re-run production services:
+
+   .. code-block:: bash
+
+      docker-compose -f docker-compose.yml -f docker-compose.production.yml  up --force-recreate
+ 
+
+To set up the mail server
+-------------------------
+
+1. Open the docker-compose.production.yml file for editing.
+2. Define the mail service. For example:
+
+   .. code-block:: yaml
+
+      mail:
+        image: modularitycontainers/postfix
+        volumes:
+          - /var/spool/postfix:/var/spool/postfix
+          - /var/mail:/var/spool/mail
+        environment:
+          MYHOSTNAME: 'mail'
+
+3. Configure the following variables in the KQueen UI service section:
+
+   .. code-block:: yaml
+
+      KQUEENUI_MAIL_SERVER: mail
+      KQUEENUI_MAIL_PORT: 10025
+
+
+Once done, you should be able to invite members through email notifications. The email will contain an activation link for the KQueen service with a possibility to set a password. Users with
+superadmin rights can also manage member roles.
+
+
+.. note::
+
+   Volume-mapping for mail containers is an additional feature that enables storing the mailing history and forwards an additional postfix mail configuration. You must properly configure it on the local machine. Otherwise, you can run the mail server without volume-mapping.
+
+To set up metrics collecting
```

Comment:
To set up metrics collecting:

Reply:
Either make it:
To set up metrics collecting:
 ~~~~~~~~~~~~~~~~~~~~

Or:

Set up metrics collecting
--------------------------------

Reply:
The same for lines 210 and 164.

Reply:
done